packages/jazz-browser/src/createBrowserContext.ts

```diff
@@ -32,15 +29,13 @@ setupInspector();
 export type BrowserContext<Acc extends Account> = {
   me: Acc;
   node: LocalNode;
-  toggleNetwork: (enabled: boolean) => void;
   logOut: () => void;
   done: () => void;
 };
 
 export type BrowserGuestContext = {
   guest: AnonymousJazzAgent;
   node: LocalNode;
-  toggleNetwork: (enabled: boolean) => void;
```

Comment:
Removed this API from the public context.

Now the anonymous localOnly switch is handled internally and if the user wants to manually control "localOnly" we re-create the context.

Done this to keep the complexity low, if we see that manual control over localOnly becomes a common pattern we can improve it.